src/uct/tcp/tcp_iface.c

```diff
@@ -255,10 +257,28 @@ static void uct_tcp_iface_connect_handler(int listen_fd, void *arg)
 
         fd = accept(iface->listen_fd, (struct sockaddr*)&peer_addr, &addrlen);
         if (fd < 0) {
+            if (errno == ECONNABORTED) {
```

Comment:
why the ucs_error on line 266 is not enough?

Reply:
removed